test/JDBC/upgrade/latest/schedule

```diff
@@ -139,8 +139,9 @@ BABEL-3646
 BABEL-383
 BABEL-405
 BABEL-937
-BABEL-3407
-forjson
+forjson-before-14-7
```

Comment:
Why do we have before-14-7 in here?

Reply:
hm, rethinking this, the name is not properly descriptive of what the tests are - these are supposed to be the regression tests that apply to both previous versions as well as any future versions. The normal forjson and forxml test suites then are only testing the new features added in 14.7. I think it would be best to go back to the original naming scheme, where these are the normal tests (aka forjson/forxml), and the other test files are named `after-14-7`

Reply:
after discussion, renaming this to forjson/forxml, and the other tests to forjson-subquery/forxml-subquery